### DISCUSSION TITLE
just want to say I really like textual and rich, sreenshot included

### DISCUSSION THREAD

**Discussion by matmaer** [Textualize/textual#5026] [Show and tell]
My mind is honestly blown by the approach of Textual and Rich. So much work taken out of my hands, it's like you're having a GUI in an OS without any desktop environment, add some tmux and you're on steroids.

Hope to publish a real app in the near future, I'm getting the hang of it.  I've learned a lot about Python just by using the Textual and Rich packages.

Random screenshot of my features testing "app", cheers!

![Screenshot From 2024-09-20 22-30-38](https://github.com/user-attachments/assets/1055ea4c-75fa-49a8-8b89-64527ffdd47e)

Big thank you to Rodigo for the christal clear videos also!

**Comment by willmcgugan** [comment]
Awesome. You might want to join the Discord server if you haven't already!

**Reply by matmaer** [reply]
Joined the server ^^
